### ISSUE TITLE
\Omega is not rendered

### ISSUE THREAD

**Issue by zcysxy** [artisticat1/obsidian-tikzjax#2]
A super weird bug 🤣: `\Omega` is not rendered. A minimal working example:

```latex
\begin{document}
\begin{tikzpicture}
\draw (0,0) node {$x\omega\Omega\Gamma$};
\end{tikzpicture}
\end{document}
```

The result is $x\omega\Gamma$, the $\Omega$ is missing 😂

**Comment by artisticat1** [comment]
Weird! I'll try to look into this when I have time.

**Comment by artisticat1** [comment]
Do you know whether this happens with any other symbols?

**Comment by zcysxy** [comment]
I tested common greek letters and math symbols, and found another one: `\otimes`.

Unlike wrong syntax or unsupported packages like `\mathbb`, which break the output image, `\Omega` and `\otimes` take up some space but are not rendered.

<img width="326" alt="image" src="https://user-images.githubusercontent.com/48207708/170855006-e019bf7a-b750-493e-8ef8-65f8e4b51b48.png">

**Comment by artisticat1** [comment]
Thanks, that was useful info!

It turns out that \Omega and \otimes were represented by the "soft hyphen" character in unicode, which is invisible. I came up with a hacky workaround where I patch the fonts and move the soft hyphen glyph to a different character.

This is now solved in 0.4.0.
